### DISCUSSION TITLE
data-bs-offset="50" is not working

### DISCUSSION THREAD

**Discussion by sameerparmar** [twbs/bootstrap#35654] [Q&A]
Hello All,

I am working on bootstrap 5 with Scrollspy component at that  data-bs-offset is not working properly.
Examplte is in below link.

https://www.tutorialrepublic.com/codelab.php?topic=bootstrap&file=adding-scrollspy-via-data-attributes

**Comment by Arhell** [comment]
https://getbootstrap.com/docs/5.1/components/scrollspy/
